### PR TITLE
numbers instead of symbols for notes

### PR DESCRIPTION
**Pull request by MAETempels**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -116,26 +116,26 @@ title: Whiting index
 
 |     | A | B | C | D | E | F | G | H |
 |-----|---|---|---|---|---|---|---|---|
-|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | ? [t][t]<br>[![](w/page219a.gif)][P219] |
+|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | N1 [t][t]<br>[![](w/page219a.gif)][P219] |
```
Comment (jo-pol) [inline docs/index.md]: I now realize these notes are not links and a comma is missing. 

```suggestion
|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | [#N1], [t][t]<br>[![](w/page219a.gif)][P219] |
```

File: docs/index.md [hunk 2]
```diff
@@ -290,24 +290,24 @@ title: Whiting index
 
 Notes on the image captions
 ---------------------------
-* **`?`**
+* **N1**
```
Comment (jo-pol) [inline docs/index.md]: Same changes for each note

```suggestion
### N1
```

**Review by jo-pol — COMMENTED** [review]
Don't know yet how to involve the others in this review process. Perhaps a bit too much at once for the moment.

File: docs/index.md [hunk 3]
```diff
@@ -116,26 +116,26 @@ title: Whiting index
 
 |     | A | B | C | D | E | F | G | H |
 |-----|---|---|---|---|---|---|---|---|
-|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | ? [t][t]<br>[![](w/page219a.gif)][P219] |
+|  1  | [diagr][A1], [c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [t][t] [![](w/page198a.gif)][P198] | [N1](#N1), [t][t]<br>[![](w/page219a.gif)][P219] |
```
Comment (jo-pol) [inline docs/index.md]: The note N5 appears nowhere in the table. Perhaps like this:

```suggestion
|  1  | [diagr][A1], [N5]:[c][c] [![](w/page70a.gif)][P70] | [diagr][B1], [N5]:[t][t] [![](w/page94a.gif)][P94] | [diagr][C1], [N5]:[t][t] [![](w/page114a.gif)][P114] | [diagr][D1], [N5]:[t][t] [![](w/page134a.gif)][P134] | [diagr][E1], [N5]:[t][t] [![](w/page155a.gif)][P155] | [diagr][F1], [t][t] [![](w/page177a.gif)][P177] | [diagr][G1], [N5]:[t][t] [![](w/page198a.gif)][P198] | [N1](#N1), [N5]:[t][t]<br>[![](w/page219a.gif)][P219] |
```

See also the other comment.

File: docs/index.md [hunk 4]
```diff
@@ -290,24 +290,24 @@ title: Whiting index
...
   The ground uses sewings, three-pair stitches, odd number of threads, tallies and/or picots.
   None of these are supported by GroundForge.
 
-* **#**
+### N4
 
   Bandages: dull for the objective of this page and too elaborate.
   
-* **`x`** short links
+### N5 short links
```
Comment (jo-pol) [inline docs/index.md]:
```suggestion
### N5
```
Also start the first phrase with something like: "This note is about the links after `[N5]:`, they" or use your own words to explain these links.

**Review by jo-pol — COMMENTED** [review]
Indeed better than the cryptic characters. These characters may have only meaning for my self and are sometimes far fetched.

File: docs/index.md [hunk 5]
```diff
@@ -412,6 +412,9 @@ a different working order is required.
   This will also cut the diagram definition by half.
   See for example "`The book of bobbin lace stitches`" by Bridget M. Cook and Geraldine Stott. 
 
+### H16 variations
```
Comment (jo-pol) [inline docs/index.md]: Whoops, forgot the dash.  So far these sections are in numerical order, it might be better to have shortest first.

Comment (jo-pol) [reply docs/index.md]:
```suggestion
### <a name="h18-variants"> H16  &mdash; variations
```

Comment (MAETempels) [reply docs/index.md]: H16 or H18?

**Pull request merged**